<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>PRAPI</title>
 
<link rel="stylesheet" href="css/main.css" type="text/css" />
<link rel="stylesheet" href="css/tr.css" type="text/css" />
<script src="js/main.js" type="text/javascript"></script>
</head>

<body id="index" class="home">
 
	<header id="banner" class="body">
		<h1><a href="index.php">PRAPI</a></h1>
		<nav>
		  <ul>
			<li ><a href="index.html">Home</a></li>
			<li><a href="manual_v2.html">Manual</a></li>
			<!--<li><a href="downloads.php">Downloads</a></li>-->
			<li><a href="example.html">Functional examples</a></li>
			<li><a href="tools.html">Relevant tools</a></li>
<!--
			<li><a href="primer_design.html">Primer design</a></li>
-->
			<li class="active"><a href="help.html">FAQ</a></li>
		  </ul>
		</nav>
		
	<section id="content" class="body">
		<div id="text" class="body">
			 
			<h2>FAQ</h2>
			<a href="help.php#sec1">Q: Can common RNA-Seq be used for  circle RNA identification and differential expression analysis?</a><br>
			
			<a href="help.php#sec8">Q: What's the differents between PRAPI  and other tools in terms of alternative splicing identifition?</a><br>
			
			<a href="help.php#sec2">Q: Can common non-stranded RNA-seq be used for NAT analysis?</a><br>
			
			<a href="help.php#sec3">Q: How to construct PAS-seq library for APA analysis?</a><br>
			
			<a href="help.php#sec4">Q: Do I need to correct PacBio reads by ourselves ?</a><br>
			
			<a href="help.php#sec5">Q: Is there a minimum number of replicates required for differential expression analysis of AS/APA?</a><br>
			
			<a href="help.php#sec6">Q: Where is the latest version of PRAPI available?</a><br>
			
			<a href="help.php#sec7">Q: I have further comments/questions. Where can I post them?</a><br>
			
			<br><br>
			<li id="sec1">
				<b>Q: Can common RNA-Seq be used for  circle RNA identification and differential expression analysis?</b><br><br>
A: Circular RNA (circRNA) is mainly generated by the splice donor of a downstream exon joining to an upstream splice acceptor, a phenomenon known as back-splicing. RNase R treatment library or rRNA-depletion RNA-Sequencing (RibominusSeq) can be used for circle RNA identification and differential expression analysis. The availability of above non-polyadenylated transcriptomes data has facilitated the genome-wide identification of thousands of circRNAs. 
			</li>
			<br>
			
			
			
			<li id="sec8">
				<b>Q: What's the differents between PRAPI  and other tools in terms of alternative splicing identifition</b><br><br>
A: we tested several of them, including TAPIS (Abdel-Ghany et al., 2016), rMATS (Shen et al., 2014), 
and ASTALAVISTA (Foissac and Sammeth, 2007). 
The venn diagram showed that most of the results were overlapped.
<br><br>
<div align="center"><img src="images/vnn.png"></div>
However，rMATS and ASTALAVISTA do not limit the strand direction during the identification of AS events. PRAPI requires that isoforms have the same transcript orientation during pair-wise comparison.
For the other tools, TAPIS (Abdel-Ghany et al., 2016), identify AS events based on only third generation sequencing, which makes it the natural point of reference tool for our comparisons. However, TAPIS excluded AS events at 5' or 3'-end flanking region. The public evidence suggests that this kind of AS type is real, thus PRAPI includes this AS type at present version.
<br><br>
<div align="center"><img src="images/vs.png"></div>


					<br>
					<br>
					

					<br>
			</li>
			
			
			<li id="sec2">
				<b>Q: Can common non-stranded RNA-seq be used for NAT analysis?</b><br><br>
A: Natural antisense transcripts (NATs) are a group of RNAs encoded within a cell that have transcript complementarity to other RNA transcripts. Thus strand-specific RNA–Seq are required for the analysis of  NAT. At present dUTP and SMART are two popular methods for construction of strand-specific RNA–Seq. The RNA was sheared to construct two shotgun and paired-end sequencing libraries following the SMART library protocol (Levin et al., 2010). Briefly, the first-strand cDNAs were synthesized using SMARTScribe reverse transcriptase with an oligonucleotide comprising the first Illumina adaptor with a random hexamer at the 5’ end after the first-strand cDNA synthesis. The first Illumina adaptors were added, and three nontemplate cytosine nucleotides were added at the 3’ end of the first strand cDNA. Then, double-stranded cDNAs were synthesized using an oligonucleotide that contained a second Illumina adaptor with three guanine ribonucleotides.
<br><br>
<b>Levin, J.Z., Yassour, M., Adiconis, X., Nusbaum, C., Thompson, D.A., Friedman, N., Gnirke, A. and Regev, A</b>. (2010) Comprehensive comparative analysis of strand-specific RNA sequencing methods. <i>Nature methods</i>, 7, 709-715.
					<br>
					<br>
					

					<br>
			</li>
			
			<br>
			<li id="sec3">
				<b>Q: How to construct PAS-seq library for APA analysis?</b><br><br>
A: Poly(A) site sequencing can be constructed by following our previous paper (<b>Wang, T., et al. 2017，Zhang Y et al. 2015</b>). Briefly, poly(A) RNAs were purified using an mRNA purification kit (Invitrogen), and heat fragmented at 95°C for 30 min. Reverse transcription (Superscript, Invitrogen) was carried out using our modified HITS-3’ adaptor at 42°C for 30 min, then the HITS-5’ adaptor (a SMART oligo) was added and incubated for an additional 30 min. The cDNAs were purified using a Qiagen PCR Cleanup kit and the second strand cDNAs were synthesized by 3 cycles of PCR using Phusion DNA polymerase (NEB) and the PE1.0 and PE2.0 primers. The PCR products were separated on a 2% agarose gel and the 200-300 bp bands were excised and purified. Gel-extracted DNAs were further amplified by a 13-cycle PCR. The PCR products were purified using a Qiagen PCR Cleanup kit. All tags were reverse-complemented because the tags were sequenced from 3’-end of transcripts. User donot need to do this transfer step since our PRAPI do the entire step for the user. User only needs to provide their original fastq files.
<br>
<br><b>Wang, T., et</b> al. Comprehensive profiling of rhizome‐associated alternative splicing and alternative polyadenylation in moso bamboo (Phyllostachys edulis). <i>The Plant Journal</i> 2017:91(4):684-699.
<br><b>Zhang Y et al</b>. Integrative genome-wide analysis reveals HLP1, a novel RNA-binding protein， regulates plant flowering by targeted alternative polyadenylation .<i>Cell Research</i>. 2015 Jun 23. 
			</li>
			
			
			<br>
			
			<li id="sec4">
				<b>Q: Do I need to correct PacBio reads by ourselves ?</b><br><br>
A: At present our PRAPI do nod performance the PacBio Long Read error correction. LSC (https://www.healthcare.uiowa.edu/labs/au/LSC/) is one wonderful software for the PacBio Long Read error correction. 
			</li>
			<br>
			
			<li id="sec5">
				<b>Q: Is there a minimum number of replicates required for differential expression analysis of AS/APA?</b><br><br>
A: Differential analysis requires of at least three replicates per sample.
			</li>
			<br>
			
			<li id="sec6">
				<b>Q: Where is the latest version of PRAPI available?</b><br><br>
A:http://www.bioinfor.org/tool/PRAPI/downloads.php always has the link to the latest version of PRAPI.
			</li>
			<br>
			
			<li id="sec7">
				<b>Q: I have further comments/questions. Where can I post them?</b><br><br>
A: You can post comments or questions to lfgu@fafu.edu.cn or yubanggaofafu@gmail.com.
			</li>
			<br>
			
			
			<h2>Suggestion or feedback</h2>
			<li>Please report misbehaviour and suggest further modifications/upgrades to 
		    yubanggaofafu@gmail.com.
		    When reporting a bug, please include all the information necessary to reproduce
		    it (operating system, browser type/version, input data).
			</li>
			<li>To contact us for more information about PRAPI: lfgu@fafu.edu.cn or yubanggaofafu@gmail.com. <br/></li>
			
		</div>
	</section>
	
	<footer id="footer" class="body">
	<img src="images/footer1.png" alt="mdc bimsb logo" class="photo" />
	<p>Copyright (c) Fujian Agriculture and Forestry University,Fuzhou 350002, China  All suggestions are welcome to lfgu@fafu.edu.cn or yubanggaofafu@gmail.com
<p
<!-- Start of StatCounter Code for Default Guide -->
<!-- Start of StatCounter Code for Default Guide -->
</p>
	</footer>
</body>
</html>

