<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>PRAPI</title>
 
<link rel="stylesheet" href="css/main.css" type="text/css" />
<link rel="stylesheet" href="css/tr.css" type="text/css" />
<script src="js/main.js" type="text/javascript"></script>
</head>

<body id="index" class="home">
 
	<header id="banner" class="body">
		<h1><a href="index.php">PRAPI</a></h1>
		<nav>
		  <ul>
			<li ><a href="index.html">Home</a></li>
			<li class="active"><a href="manual_v2.html">Manual</a></li>
			<!--<li><a href="downloads.php">Downloads</a></li>-->
			<li><a href="example.html">Functional examples</a></li>
			<li><a href="tools.html">Relevant tools</a></li>
<!--
			<li><a href="primer_design.html">Primer design</a></li>
-->
			<li><a href="help.html">FAQ</a></li>
		  </ul>
		</nav>
<!--
		<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?63c9a8f0fd600bd423806ed37103e069";
  var s = document.getElementsByTagName("script")[0]; 
  s.parentNode.insertBefore(hm, s);
})();
</script>
-->
	</header>
	<section id="content" class="body">
		<div id="text" class="body">
			<h2>Contents</h2>
			
			<ol>
			<ul>
				<li><a href="manual.php#sec1">Install</a></li>
				<li><a href="manual.php#sec2">Usage</a></li>
				<li><a href="manual.php#sec3">Download and Run the Example data</a></li>
				<li><a href="manual.php#sec4">Interpreting PRAPI output</a></li>
			</ul>
			</ol>
			
			<h3 id="sec1">1.Install</h3>
			</br>
			<pre>
wget http://forestry.fafu.edu.cn/tool/PRAPI/prapi_env.yaml
conda env create -f prapi_env.yaml #install conda environment
conda activate prapi_env #activate env 
pip install -i https://pypi.anaconda.org/gaoyubang/simple splicegrapher #install depend package
			</pre>
</pre>
			<h3 id="sec2">2.Usage</h3>
			<p>If you want to take DE_APA analysis,please make sure that the bam file is generated by
				PAS_Seq reads.If you want to take Circle analysis,please make sure that the bam file is generated by
				RNase R easeing reads . Before running the script, you will need to run gmap_build to make a genome reference index.
				Please make sure that the section info you want is right.</p>
			<pre>
$Pacbio_v16.py -c conf.txt
All parameter is stored in conf.txt.
basic arguments
Output_dir          Name of the output directory
Pacbio_reads       PacBio sequence's full path and name
GMAP_IndexesDir     Directory of genomic index files buided by gmap_build program
GMAP_Process        Number of worker threads for GMAP
Genome_Annotion     Reference annotation  in gpd format <a href="https://genome.ucsc.edu/FAQ/FAQformat.html#format9">https://genome.ucsc.edu/FAQ/FAQformat.html#format9 </a>
MaxIntron           Max length for one internal intron (default 200000)
Multile_processing  Using parallel version
other arguments
MinDist             Minimum distance between any two poly(A) or transcripts start sites
MinSupport          Minimum number of trusted reads supporting a poly(A) or transcripts
start sites
Width_of_peaks      Peak widths for searching poly(A)
Graph arguments
Group              Groups of different library type to specify ylim for each facet separately
anchorLength        Min anchor Length for Tophat/Bowtie aligner
DElib               The libraries used for differential analysis
P value        (default 0.01)
FDR               (default 0.01)
			</pre>
			<h3 id="sec3">3.Download and Run the Example data</h3>
			<a href="download/v2/test_v2.tar.gz">Click to Download example data</a>
			<pre>
$ wget http://forestry.fafu.edu.cn/tool/PRAPI/download/v2/test_v2.tar.gz -O - | tar xvzf - 
$ cd test_v2 
$ sh run.sh
			</pre>
			<h3 id="sec4">4.Interpreting PRAPI output</h3>
			<a href="download/v2/test_v2.tar.gz">Click to Download test output</a>
			<pre>
<b>tree test_result</b>
test_v2/*/output
|-- Annotation_miss
|   |-- Miss_annotation.csv
|-- APA
|   |-- apa.txt_gene.csv
|   |-- de_apa.txt.gene.fdr.csv
|-- AS
|   |-- as.A3SS.txt_gene.csv
|   |-- as.A5SS.txt_gene.csv
|   |-- as.RI.txt_gene.csv
|   |-- as.SE.txt_gene.csv
|-- AS_cufflinks
|   |-- as.A3SS.txt_gene.csv
|   |-- as.A5SS.txt_gene.csv
|   |-- as.RI.txt_gene.csv
|   |-- as.SE.txt_gene.csv
|-- JC
|   |-- A3SS.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|   |-- A5SS.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|   |-- RI.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|   |-- SE.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|-- JC_cuff
|   |-- A3SS.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|   |-- A5SS.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|   |-- RI.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|   |-- SE.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|-- AS_IDP
|   |-- as.A3SS.txt_gene.csv
|   |-- as.A5SS.txt_gene.csv
|   |-- as.RI.txt_gene.csv
|   |-- as.SE.txt_gene.csv
|-- JC_IDP
|   |-- A3SS.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|   |-- A5SS.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|   |-- RI.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|   |-- SE.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv
|-- ATI
|   |-- ati.txt_gene.csv
|-- NAT
|   |-- de_nat.txt_gene.csv
|-- circle
|   |-- Lib*_circ.txt
|   |-- Lib*_VS_Lib*.txt
|   |-- Lib*_degseq_Lib*
|       |-- output_score.txt
|-- Graph
|   |-- Annotation_Gene
|       |-- Gene1.pdf
|       |-- ...
|       |-- GeneN.pdf
|   |-- Novel_Gene
|       |-- Gene1.pdf
|       |-- ...
|       |-- GeneN.pdf
|-- Novel_Gene
|       |-- Novel_Gene.fa

</pre>
<li> Annotation_miss - Annotation correction contains annotation correction gene.</li>
Miss_annotation.csv format is below.
<table  class="gridtable">
	<tr><th>Gene_name</th><th>Name of annotation correction gene.</th></tr>
	<tr><th>Strand</th><th>strand of annotation correction gene.</th></tr>
	<tr><th>Support_Reads_Num</th><th>Reads number of Pacbio that support this annotation correction gene.</th></tr>
	<tr><th>Support_Reads</th><th>Pacbio Reads that support this annotation correction gene.</th></tr>
</table>
<li> APA - contains poly(A) site predicted and differential APA.</li>
apa.txt_gene.csv format is below.
<table  class="gridtable">
	<tr><th>Gene_name</th><th>Name of genes that have Alternative cleavage and APA site.</th></tr>
	<tr><th>Reads</th><th>Pacbio Reads that support this Alternative cleavage and APA site.</th></tr>
	<tr><th>Pos</th><th>Genome coordinate of Alternative cleavage and APA site.</th></tr>
</table>
de_apa.txt.gene.fdr.csv format is below.
<table  class="gridtable">
	<tr><th>Gene_name</th><th>Name of genes that have differential APA site.</th></tr>
	<tr><th>Chromosome</th><th>Chromosome of this gene.</th></tr>
	<tr><th>Lib</th><th>Libs pairs of this genes which have differential APA site.</th></tr>
	<tr><th>Pos</th><th>Genome coordinate of differential APA site.</th></tr>
	<tr><th>Pvalue</th><th>Pvalue of this differential APA site.</th></tr>
	<tr><th>FDR</th><th>FDR of this differential APA site</th></tr>
</table>
<li> AS - contains four major type of AS file.</li>
as.*.txt_gene.csv format is below.
<table  class="gridtable">
	<tr><th>Gene_name</th><th>Name of genes that have AS.</th></tr>
	<tr><th>GeneID</th><th>Default values.</th></tr>
	<tr><th>geneSymbol</th><th>Default values.</th></tr>
	<tr><th>chr</th><th>Chromosome of this gene.</th></tr>
	<tr><th>strand</th><th>Strand of this gene.</th></tr>
	<tr><th>The remaining 6 columns</th><th>The upstream and downstream info of this AS events.</th></tr>
</table>
<li> JC - contains four major type of differential AS file in Pacbio reads.</li>

*.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv format is below.
<table  class="gridtable">
	<tr><th>Gene_name</th><th>Name of genes that have AS.</th></tr>
	<tr><th>GeneID</th><th>Default values.</th></tr>
	<tr><th>geneSymbol</th><th>Default values.</th></tr>
	<tr><th>chr</th><th>Chromosome of this gene.</th></tr>
	<tr><th>strand</th><th>Strand of this gene.</th></tr>
	<tr><th>The remaining 6 columns</th><th>The upstream and downstream info of this AS events.</th></tr>
	<tr><th>IC_SAMPLE_1</th><th>Inclusion counts for SAMPLE_1, replicates are separated by comma.</th></tr>
	<tr><th> SC_SAMPLE_1</th><th>Skipping counts for SAMPLE_1, replicates are separated by comma.</th></tr>
	<tr><th>IC_SAMPLE_2</th><th>Inclusion counts for SAMPLE_2, replicates are separated by comma.</th></tr>
	<tr><th> SC_SAMPLE_2</th><th>Skipping counts for SAMPLE_2, replicates are separated by comma.</th></tr>
	<tr><th>IncFormLen</th><th>length of inclusion form, used for normalization.</th></tr>
	<tr><th>SkipFormLen</th><th>length of skipping form, used for normalization.</th></tr>
	<tr><th>IncLevel1</th><th>Inclusion level for SAMPLE_1 replicates (comma separated) calculated from normalized counts.</th></tr>
	<tr><th>IncLevel2</th><th>Inclusion level for SAMPLE_2 replicates (comma separated) calculated from normalized counts.</th></tr>
	<tr><th>IncLevelDifference/th><th>average(IncLevel1) - average(IncLevel2).</th></tr>
	<tr><th>FDR</th><th>FDR of this differential AS.</th></tr>
	<tr><th>PValue</th><th>Pvalue of this differential AS.</th></tr>
</table>
<li> AS_cuff - contains four major type of AS file in Pacbio reads in NGS assembly.</li>
as.*.txt_gene.csv format is as AS.
<li> JC_cuff - contains four major type of differential AS file in NGS assembly.</li>
*.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv format is as JC.
<li> AS_IDP - contains four major type of AS file in NGS by IDP predict and identify isoforms.</li>
as.*.txt_gene.csv format is as AS.
<li> JC_IDP - contains four major type of differential AS file by IDP predict and identify isoforms.</li>
*.MATS.ReadsOnTargetAndJunctionCount.txt_gene.csv format is as JC.
<li> ATI - contains files of trascripts start site.</li>
ati.txt_gene.csv format is below.
<table  class="gridtable">
	<tr><th>Gene_name</th><th>Name of genes that have ATI site.</th></tr>
	<tr><th>Reads</th><th>Pacbio Reads that support this ATI site.</th></tr>
	<tr><th>Pos</th><th>Genome coordinate of ATI site.</th></tr>
</table>
<li> NAT - contains files of NAT and differential NAT.</li>
nat.txt_gene.csv format is below.
<table  class="gridtable">
	<tr><th>Gene_name</th><th>Name of genes that have NATs.</th></tr>
	<tr><th>Plus_Reads</th><th>Plus strand Reads in NATS.</th></tr>
	<tr><th>Minus_Reads</th><th>Minus strand Reads in NATS.</th></tr>
</table>
de_nat.txt_gene.csv format is below.
<table  class="gridtable">
	<tr><th>Gene_name</th><th>Name of genes that have NATs.</th></tr>
	<tr><th>logFC</th><th>Log-fold-changes between each pair of NATs.</th></tr>
	<tr><th>logCPM</th><th>Log2 counts-per-million of NATs.</th></tr>
	<tr><th>PValue</th><th>Pvalue of this differential NATs.</th></tr>
	<tr><th>FDR</th><th>FDR of this differential NATs.</th></tr>
</table>
<li> circle - contains files of circle libs and differential circle files.</li>
Lib*_circ.txt format is below.
<table  class="gridtable">
	<tr><th>start</th><th>Start of junction.</th></tr>
	<tr><th>end</th><th>End of junction.</th></tr>
	<tr><th>name</th><th>Circular RNA/Junction reads.</th></tr>
	<tr><th>score</th><th>Flag to indicate realignment of fusion junctions.</th></tr>
	<tr><th>strand</th><th>+ or - for strand.</th></tr>
	<tr><th>thickStart</th><th>No meaning.</th></tr>
	<tr><th>thickEnd</th><th>No meaning.</th></tr>
	<tr><th>itemRgb</th><th>0,0,0.</th></tr>
	<tr><th>exonCount</th><th>Number of exons.</th></tr>
	<tr><th>exonSizes</th><th>Exon sizes.</th></tr>
	<tr><th>exonOffsets</th><th>Exon offsets.</th></tr>
	<tr><th>readNumber</th><th>Number of junction reads</th></tr>
	<tr><th>circType</th><th>'circRNA' or 'ciRNA'.</th></tr>
	<tr><th>gene_Name </th><th>Name of gene.</th></tr>
	<tr><th>isoformName </th><th>Name of isoform.</th></tr>
	<tr><th>exonIndex/intronIndex </th><th>Index (start from 1) of exon (for circRNA) or intron (for ciRNA) in given isoform.</th></tr>
	<tr><th>flankIntron </th><th>Left intron/Right intron.</th></tr>
</table>
Lib*_VS_Lib*.txt format is below.
<table  class="gridtable">
	<tr><th>Circle information</th><th>gene name of this circle|Chromosome of this gene:circle start pos-circle end pos.</th></tr>
	<tr><th>Count number in lib1</th><th>Count number of circle in lib1.</th></tr>
	<tr><th>Count number in lib2</th><th>Count number of circle in lib2.</th></tr>
</table>
output_score.txt format is below.
<table  class="gridtable">
	<tr><th>GeneNames</th><th>gene name of this circle|Chromosome of this gene:circle start pos-circle end pos.</th></tr>
	<tr><th>value1</th><th>Count number of circle in lib1.</th></tr>
	<tr><th>value2</th><th>Count number of circle in lib2.</th></tr>
	<tr><th>log2(Fold_change)</th><th>log2(Fold_change) of Count number in lib1 and lib2.</th></tr>
	<tr><th>log2(Fold_change) normalized</th><th>log2(Fold_change) normalized of Count number in lib1 and lib2.</th></tr>
	<tr><th>z-score</th><th>z-score of de circle.</th></tr>
	<tr><th>p-value</th><th>p-value of de circle.</th></tr>
	<tr><th>q-value(Benjamini et al. 1995)</th><th>q-value(Benjamini et al. 1995) of de circle.</th></tr>
	<tr><th>q-value(Storey et al. 2003)</th><th>q-value(Storey et al. 2003) of de circle.</th></tr>
	<tr><th>Signature(p-value < 0.01)</th><th>Signature.</th></tr>
</table>
<li> Graph - contains pdf file of annotation gene and novel gene.</li>
<li> Novel_Gene - contains file of novel gene.</li>
Novel_Gene.fa format is below.
<table  class="gridtable">
	<tr><th>Chrom</th><th>Chromosome of this novel gene.</th></tr>
	<tr><th>Start</th><th>Start sites of this novel gene.</th></tr>
	<tr><th>End</th><th>End sites of this novel gene.</th></tr>
	<tr><th>Reads_Num</th><th>Pacbio reads number that support this novel gene.</th></tr>
	<tr><th>Reads_Id</th><th>Pacbio reads id that support this novel gene.</th></tr>
	<tr><th>Seq</th><th>Seqs prediction of this novel gene.</th></tr>
</table>
		</div>
	</section>
	<footer id="footer" class="body">
	<img src="images/footer1.png" alt="mdc bimsb logo" class="photo" />
	<p>Copyright (c) Fujian Agriculture and Forestry University,Fuzhou 350002, China  All suggestions are welcome to lfgu@fafu.edu.cn or yubanggaofafu@gmail.com
<p
<!-- Start of StatCounter Code for Default Guide -->
<!-- Start of StatCounter Code for Default Guide -->
</p>
	</footer>
</body>
</html>

