<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>PRAPI</title>
 
<link rel="stylesheet" href="css/main.css" type="text/css" />
<link rel="stylesheet" href="css/tr.css" type="text/css" />
<script src="js/main.js" type="text/javascript"></script>
</head>

<body id="index" class="home">
 
	<header id="banner" class="body">
		<h1><a href="index.php">PRAPI</a></h1>
		<nav>
		  <ul>
			<li ><a href="index.html">Home</a></li>
			<li><a href="manual_v2.html">Manual</a></li>
			<!--<li><a href="downloads.php">Downloads</a></li>-->
			<li><a href="example.html">Functional examples</a></li>
			<li class="active"><a href="tools.html">Relevant tools</a></li>
<!--
			<li><a href="primer_design.html">Primer design</a></li>
-->
			<li><a href="help.html">FAQ</a></li>
		  </ul>
		</nav>
	<section id="content" class="body">
		<div id="text" class="body">
			<h2>List of all kinds of wonderful tools for Iso-Seq</h2>
	<ul>
		<li>
			<a href="https://www.healthcare.uiowa.edu/labs/au/LSC">LSC</a>- Error correction of PacBio long reads.
		</li>
		<li>
			<a href="https://www.healthcare.uiowa.edu/labs/au/IDP/">IDP</a>- Isoform Detection and Prediction.
		</li>
		<li>
			<a href="https://www.healthcare.uiowa.edu/labs/au/IDP-fusion/">IDP-fusion</a>- A tool for Isoform Detection and Prediction for Fusion Genes 
from Second Generation Sequencing and PacBio sequencing.
		</li>
		<li>
			<a href="https://web.stanford.edu/group/wonglab/SpliceMap/">SpliceMap</a>- Novel splice junction discovery.
		</li>
		<li>
			<a href="https://bitbucket.org/comp_bio/tapis">TAPIS</a>- A package for prediction and analysis of Pacific BioSciences Iso-Seq data. It performs error correction, splice isoform prediction, and prediction alternative polyadenylation sites.
		</li>
		<li>
			<a href="https://github.com/goeckslab/isoseq-browser">Iso-Seq Browser</a>- An interactive visual analytics tool for long-read RNA sequencing data produced by Pacific Biosciences’ isoform sequencing (Iso-Seq) techniques.
		
		</li>	
		<li>
			<a href="https://github.com/liuxiaoxian/IsoSeq_AS_de_novo">IsoSeq_AS_de_novo</a>- All scripts are writtern for doing de novo alternative splicing (AS) event detection using Iso-Seq data.
		</li>
		<li>
			<a href="https://github.com/PacificBiosciences/SMRT-Analysis">SMRT Analysis</a>- A powerful, open-source bioinformatics software suite available for analysis of DNA sequencing data from Pacific Biosciences' SMRT technology.
		</li>	
		<li>
			<a href="http://www.atgc-montpellier.fr/lordec/">LoRDEC</a>- A hybrid error correction program for long, PacBio reads.
		</li>
		<li>
			<a href="https://github.com/alexdobin/STAR">STAR</a>- Spliced Transcripts Alignment to a Reference.
		</li>
		<li>
			<a href="http://research-pub.gene.com/gmap/">GMAP</a>- A Genomic Mapping and Alignment Program for mRNA and EST Sequences.
		</li>
		<li>
			<a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">Bowtie2</a>- An ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences.
		</li>
		<li>
			<a href="https://github.com/samtools/samtools">SAMtools</a>- A suite of programs for interacting with high-throughput sequencing data
		</li>
</ul>
			<h2>Reference</h2>
			<ul>
			<li>Abdel-Ghany, S.E., et al. A survey of the sorghum transcriptome using single-molecule long reads. <i>Nature communications</i> 2016;7.<br/></li>
			<li>Au, K.F., et al. Characterization of the human ESC transcriptome by hybrid sequencing. <i>Proceedings of the National Academy of Sciences</i> 2013;110(50):E4821-E4830.<br/></li>
			<li>Gao, Y., Wang, J. and Zhao, F. CIRI: an efficient and unbiased algorithm for de novo circular RNA identification. <i>Genome biology </i>2015;16(1):4.	<br/></li>
			<li>Li, H., et al. The sequence alignment/map format and SAMtools. <i>Bioinformatics</i> 2009;25(16):2078-2079.	<br/></li>
			<li>Robinson, M.D., McCarthy, D.J. and Smyth, G.K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. <i>Bioinformatics</i> 2010;26(1):139-140.<br/></li>
			<li>Shen, S., et al. rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-Seq data. <i>Proceedings of the National Academy of Sciences</i> 2014;111(51):E5593-E5601.<br/></li>
			<li>Wang, T., et al. Comprehensive profiling of rhizome‐associated alternative splicing and alternative polyadenylation in moso bamboo (Phyllostachys edulis). <i>The Plant Journal</i> 2017.	<br/></bi>
			<li>Wu, T.D. and Watanabe, C.K. GMAP: a genomic mapping and alignment program for mRNA and EST sequences. <i>Bioinformatics</i> 2005;21(9):1859-1875.<br/></li>
			</ul>
		</div>
	</section>
	
	<footer id="footer" class="body">
	<img src="images/footer1.png" alt="mdc bimsb logo" class="photo" />
	<p>Copyright (c) Fujian Agriculture and Forestry University,Fuzhou 350002, China  All suggestions are welcome to lfgu@fafu.edu.cn or yubanggaofafu@gmail.com
<p
<!-- Start of StatCounter Code for Default Guide -->
<!-- Start of StatCounter Code for Default Guide -->
</p>
	</footer>
</body>
</html>
